src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -171,7 +171,6 @@ private function getPathOperation(string $operationName, array $operation, strin
             case 'DELETE':
                 return $this->updateDeleteOperation($pathOperation, $resourceShortName);
             default:
```

Comment:
this line should be reverted and the `return` statement on the last line added back.